frontend/www/js/omegaup/components/problem/Tags.vue

```diff
@@ -195,6 +222,9 @@ import { types } from '../../api_types';
 @Component
 export default class ProblemTags extends Vue {
   @Prop() initialTags!: omegaup.Tag[];
+  @Prop() problemLevel!: string;
```

Comment:
```suggestion
  @Prop({ default: null }) problemLevel!: string | null;
```

welp, se me fue esto en la ronda anterior. ojalá Vue 3 y/o alguna nueva versión de los decoradores nos ayuden a atrapar esta clase de errores.